external/PATCH-01-STRATUM

```diff
@@ -3829,32 +3837,29 @@ index 9f4bb3a9..e0154d94 100644
 +}
 +
 +::util::Status TdiSdeWrapper::AddPort(int device, int port, uint64 speed_bps,
-                                      FecMode fec_mode) {
++                                     FecMode fec_mode) {
 +  auto port_attrs = absl::make_unique<port_attributes_t>();
 +#ifdef P4SDE_DPDK_TARGET_STUB
-   ASSIGN_OR_RETURN(auto bf_speed, PortSpeedHalToBf(speed_bps));
-   ASSIGN_OR_RETURN(auto bf_fec_mode, FecModeHalToBf(fec_mode, speed_bps));
--  RETURN_IF_BFRT_ERROR(bf_pal_port_add(static_cast<bf_dev_id_t>(device),
++  ASSIGN_OR_RETURN(auto bf_speed, PortSpeedHalToBf(speed_bps));
++  ASSIGN_OR_RETURN(auto bf_fec_mode, FecModeHalToBf(fec_mode, speed_bps));
 +#endif
 +  RETURN_IF_TDI_ERROR(bf_pal_port_add(static_cast<bf_dev_id_t>(device),
-                                        static_cast<bf_dev_port_t>(port),
--                                       bf_speed, bf_fec_mode));
++                                       static_cast<bf_dev_port_t>(port),
 +                                       port_attrs.get()));
-   return ::util::OkStatus();
- }
- 
--::util::Status BfSdeWrapper::DeletePort(int device, int port) {
--  RETURN_IF_BFRT_ERROR(bf_pal_port_del(static_cast<bf_dev_id_t>(device),
++  return ::util::OkStatus();
++}
++
 +::util::Status TdiSdeWrapper::AddPort(int device, int port, uint64 speed_bps,
 +                                     PortConfigParams& config,
-+                                     FecMode fec_mode) {
+                                      FecMode fec_mode) {
 +  static int port_in;
 +  static int port_out;
 +
 +  auto port_attrs = absl::make_unique<port_attributes_t>();
 +#ifdef P4SDE_DPDK_TARGET_STUB
-+  ASSIGN_OR_RETURN(auto bf_speed, PortSpeedHalToBf(speed_bps));
-+  ASSIGN_OR_RETURN(auto bf_fec_mode, FecModeHalToBf(fec_mode, speed_bps));
+   ASSIGN_OR_RETURN(auto bf_speed, PortSpeedHalToBf(speed_bps));
+   ASSIGN_OR_RETURN(auto bf_fec_mode, FecModeHalToBf(fec_mode, speed_bps));
+-  RETURN_IF_BFRT_ERROR(bf_pal_port_add(static_cast<bf_dev_id_t>(device),
```

Comment:
why is this being removed?

Reply:
I didnt removed this or modified the original file, git diff generated this change